keyboards/dyz/dyz60_hs/readme.md

```diff
@@ -0,0 +1,19 @@
+# dyz60
+
+![dyz60 pcb](https://i.imgur.com/l9z5KuJh.jpg)
+
+60% layout has flexcut with multiple layout and mounting support.
+
+* Keyboard Maintainer: [dayatz](https://github.com/dayatz)
+* Hardware Supported: dyz60, ATmega32U4
+* Hardware Availability: Local Indonesia GB.
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware?  This is needed, as a new user may not be aware of how to enter to bootloader to flash new firmware.

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```